docs/mkdocs/search_index.json

```diff
@@ -26,44 +26,29 @@
             "title": "System Overview"
         }, 
         {
-            "location": "/arch_overview/#system-boundaries", 
-            "text": "What does the application interact with\nWhat other systems interact with it", 
-            "title": "System Boundaries"
-        }, 
-        {
-            "location": "/arch_overview/#sub-systems", 
-            "text": "I.e. API, Admin, Clients, AWS, Email Autoscaling etc", 
-            "title": "Sub-systems"
+            "location": "/systems/#govuk-notify-systems-overview", 
+            "text": "", 
+            "title": "GOV.UK Notify Systems Overview"
         }, 
         {
-            "location": "/arch_overview/#dataflow-sequence", 
-            "text": "Use cases with flow through the system for common paths i.e. one for csv one for client etc\nMaybe for common sub-systems i.e. how does auto scaling and monitoring work", 
-            "title": "Dataflow / Sequence"
+            "location": "/systems/#system-contextboundary", 
+            "text": "Notify provides a service for Government Departments to send notifications to their end users. Hence, our 'users' are usually the Government departments rather than the end users. We provide 3 type of services: email, text messaging and letters (available soon).    This system context/boundary diagram shows where external users and systems interact with Notify's backend systems. The interaction points are as follow:", 
```

Comment:
Suggest changing "The interaction points are as follow" to "The interaction points are as follows"